src/ORM/DB.php

```diff
@@ -696,4 +704,100 @@ public static function alteration_message($message, $type = "")
     {
         self::get_schema()->alterationMessage($message, $type);
     }
+
+    /**
+     * Check if all tables and field columns for a class exist in the database.
+     *
+     * @param string $class
+     * @return boolean
+     */
+    public static function databaseIsReady(string $class): bool
```

Comment:
This is a bit of a misleading method name since it's for a specific dataobject, rather than the entire database.  Maybe rename it to something like `tablesCreated(string $class): bool`.  Also update the protected static property to match

Reply:
That's a fair point. `tablesCreated` isn't quite right, since it's checking more than just if the tables are ready, but you're right that the current name isn't right either. I'll think of something more accurately descriptive.